### PR TITLE
Create guildMemberUpdate.js

### PR DESCRIPTION
**Pull request by tovade**

### CODE REVIEW

File: src/events/guildMemberUpdate.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const { getAuditChannel } = require("../utils/functions");
+const { MessageEmbed } = require('discord.js');
+const Discord = require('discord.js')
```
Comment (casperiv0) [inline src/events/guildMemberUpdate.js]: Could you please remove this line as it's not being used.

File: src/events/guildMemberUpdate.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    name: "guildMemberUpdate",
+    async execute(bot, newMember, oldMember) {
+            const auditChannel = await getAuditChannel(newMember.guild.id);
+        
+            // not enabled
+            if (auditChannel === null || !auditChannel) return;
+        const embed = new MessageEmbed()
+    .setAuthor(`${newMember.user.tag}`, newMember.user.displayAvatarURL({ dynamic: true }))
+    .setTimestamp()
+    .setColor(oldMember.guild.me.displayHexColor);
```
Comment (casperiv0) [inline src/events/guildMemberUpdate.js]: Can you update the color to `ORANGE` to keep consistency across the other update events

**Review by casperiv0 — CHANGES_REQUESTED** [review]
Please view requested. Do you mind to format the code too :) please, thanks!

File: src/events/guildMemberUpdate.js (hunk 2 above)
Comment (tovade) [reply src/events/guildMemberUpdate.js]: > Please view requested. Do you mind to format the code too :) please, thanks!

what?
i dont get your question?

**Comment by tovade** [comment]
i updated the color and removed the unused line

**Comment by casperiv0** [comment]
Looks good! Thankyou for creating this pull request 🔥

**Pull request merged**